css/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  font: garamond;
```

Comment:
@AmyMirisola A fallback font should be used incase one of the fonts cannot load for some reason.  So inthis instance maybe use `font-family: garound, georgia, serif`. `font` by itself is a sorcut to writting multiple font relate declarations in one line. `font: 14px red italic serif` wher the font-size and font family are required.